docs/running-tests.rst

```diff
@@ -12,7 +12,7 @@ virtual environment.
 .. code-block:: text
 
     # Install python-ldap dependencies
-    sudo dnf install -y gcc python3-devel openldap-devel
+    sudo dnf install -y gcc python3-devel openldap-devel libssh libssh-devel
```

Comment:
The added packages are dependencies for pylibssh thus the comment above is misleading.